raptorWeb/staffapps/models.py

```diff
@@ -0,0 +1,98 @@
+from django.db import models
```

Comment:
General question/suggestion for a possibly better system. In what ways could we only use a single Model `StaffApplication` for all forms, and still allow different questions and different types of forms that still behave the same way for the end user?